src/components/learner-credit-management/data/utils.js

```diff
@@ -95,6 +95,16 @@ export const transformUtilizationTableResults = results => results.map(result =>
   courseKey: result.courseKey,
 }));
 
+export const transformUtilizationTableTransactionResults = results => results.map(result => ({
+  created: result.created,
+  enterpriseEnrollmentId: result.fulfillmentIdentifier,
+  userEmail: result.lmsUserEmail,
+  courseTitle: result.contentTitle,
+  courseListPrice: result.unit === 'usd_cents' ? -1 * (result.quantity / 100) : -1 * results.quantity,
+  uuid: uuidv4(),
```

Comment:
[curious/question] Should this be relying on the `uuid` [returned by the `transactions` API](https://enterprise-subsidy.edx.org/api/schema/redoc/#tag/transactions/operation/api_v2_subsidies_admin_transactions_list)?

Reply:
yup another copy pasta artifact, good catch